src/view/app.rs

```diff
@@ -280,12 +280,33 @@ fn get_picker() -> Option<Picker> {
         })
         .filter(|picker| picker.protocol_type != ProtocolType::Halfblocks)
 }
-
 #[cfg(target_os = "windows")]
 fn get_picker() -> Option<Picker> {
-    // Todo! figure out how to get the size of the terminal on windows
-    // I think with the winapi it is possible
-    let mut picker = Picker::new((10, 17));
+    use windows_sys::Win32::System::Console::GetConsoleWindow;
+    use windows_sys::Win32::UI::HiDpi::GetDpiForWindow;
+
+    struct FontSize {
+        pub width: u16,
+        pub height: u16,
+    }
+
+    let size: FontSize = match unsafe { GetDpiForWindow(GetConsoleWindow()) } {
+        96 => FontSize {
+            width: 9,
+            height: 20,
+        },
+        120 => FontSize {
+            width: 12,
+            height: 25,
+        },
+        144 => FontSize {
+            width: 14,
+            height: 32,
+        },
+        _ => unimplemented!("Other DPIs then 96 (100%), 120 (125%) and 144 (150%) are supported as of now"),
```

Comment:
I assume this case is rare, panicking leaves the application unusuable

Reply:
That's very rare, yeah, but it will be an issue for those people. I guess I can make a default value for this so they can at least use it as downloader